package-utils.js

```diff
@@ -0,0 +1,81 @@
+var fs = require('fs');
+var path = require('path');
+var shell = require('shelljs');
+var ncp = require('child_process');
+
+//------------------------------------------------------------------------------
+// shell functions
+//------------------------------------------------------------------------------
+var rp = function (relPath) {
+    return path.join(shell.pwd() + '', relPath);
+}
+exports.rp = rp;

```

Comment:
Have you taken these functions from vsts-tasks or other similar repo? Then it would be good to add that in comment